docs/sentry-doc-config.json

```diff
@@ -11,6 +11,14 @@
         "index#reporting-errors"
       ]
     },
+    "javascript.textkit": {
```

Comment:
This entry is used for a "Framework selection panel" when creating a new project, so we cannot really add it here

Reply:
where do you think it suites the most?

Reply:
oh... saw your comment below about the `Tips & Tricks`